internal/command/test.go

```diff
@@ -101,7 +101,27 @@ func (c *TestCommand) Run(rawArgs []string) int {
 	}
 
 	view := views.NewTest(args.ViewType, c.View)
-	var junitXMLView *views.TestJUnitXMLFile
+
+	// The specified testing directory must be a relative path, and it must
```

Comment:
This diff is being rendered in a confusing way. The change I made was to move the JUnit-related block (checking experimental status, setting up vars etc) to go underneath the block starting `config, configDiags := c.loadConfigWithTests(".", args.TestDirectory)`.  This is due to needing `c.configLoader` to be a non-nil pointer when we call `NewTestJUnitXMLFile`.